core/src/main/scala/io/finch/internal/ToResponse.scala

```diff
@@ -56,6 +58,21 @@ object ToResponse extends LowPriorityToResponseInstances {
     rep
   }
 
+  implicit def asyncStreamToResponse[A, CT <: String](implicit
```

Comment:
Hm. I guess you're right! Maybe make it new-line delimited by default? I wonder which approach is considered a default.